dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -26,6 +36,65 @@
 
 public class CommunityRestRepositoryIT extends AbstractControllerIntegrationTest {
 
+    public void createTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        ObjectMapper mapper = new ObjectMapper();
+        CommunityRest comm = new CommunityRest();
+        comm.setName("Test Top-Level Community");
```

Comment:
this is interesting...
the name should be a read-only property of the RestObject but it is not yet defined in this way.
If we allow to set the name we can have inconsistency as in your current code where the name and dc.title are different

Reply:
Yes. The behavior in DSpace 6 is to pass the title to the "name". But as you state, this isn't defined yet in that way. I suggest we finish this PR and then open a ticket to fix that specific issue.